index.src.html

```diff
@@ -519,14 +520,18 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   execute the following step:
 
   9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+      <a>`Clear-Site-Data`</a> and the request's <a>credentials mode</a> permits
+      modifying credentials, then execute [[#clear-response]] on |response|.
```

Comment:
My reasoning is that the intent of restricting credentials during `fetch()` is that "I only want to load some content, I don't want modifications to my data" so applying this to other data types seems natural. It's true that the definition of `credentials mode` only pertains to cookies, but then again, it was written at a time when there were no headers affecting `localStorage` (...at least to my knowledge?).

But I don't feel strongly either - happy to defer to you.

Regarding the other point, to make sure I understand - should I close this pull request and make a change in the fetch spec? It just didn't seem right to me to modify an established spec with an experimental feature, but if it's OK, I'll do that.